src/coreclr/tools/Common/Internal/Metadata/NativeFormat/Generator/SchemaDef.cs

```diff
@@ -315,6 +315,13 @@ internal sealed class SchemaDef
                     members: new MemberDef[] {
                         new MemberDef(name: "Value", typeName: "string")
                     }
+                ),
+                new RecordDef(
+                    name: "ConstantBoxedEnumValue",
```

Comment:
```suggestion
                    name: "ConstantEnumValue",
```
The value is not stored as boxed object instance in the metadata format. "Boxed" does not make sense to me here.

(I understand that it is a pre-existing problem.)